examples/miscellaneous/miscellaneous_ex9/augment_sparsity_on_interface.C

```diff
@@ -43,14 +43,14 @@ void AugmentSparsityOnInterface::mesh_reinit ()
             {
               std::unique_ptr<const Elem> side_elem = elem->build_side_ptr(side);
 
-              lower_centroids[std::make_pair(elem, side)] = side_elem->centroid();
+              lower_centroids[std::make_pair(elem, side)] = side_elem->vertex_average();
```

Comment:
Would it be worthwhile to search-and-replace in all the variable names too?

`centroid = elem->not_a_centroid();` is funny but it might confuse someone years down the line.

Reply:
I added a comment in the `CentroidPartitioner` docs, but did not rename the class... I think the current name is more evocative even if less correct than e.g. `ElemVertexAveragePartitioner` or something like that. Also we'd have to keep the old name around for backwards compatibility, etc. regardless.

Reply:
Yeah, I wouldn't want to change any class names, thanks to the added hassle; I'm just thinking about these little internal variables here and there.